src/GovUk.Education.ExploreEducationStatistics.Data.Services/AbstractSubjectMetaService.cs

```diff
@@ -12,12 +11,12 @@ namespace GovUk.Education.ExploreEducationStatistics.Data.Services
 {
     public abstract class AbstractSubjectMetaService
     {
-        protected readonly IFilterItemRepository _filterItemRepository;
+        protected readonly IFilterItemRepository FilterItemRepository;
         protected static IComparer<string> LabelComparer { get; } = new LabelRelationalComparer();
 
         protected AbstractSubjectMetaService(IFilterItemRepository filterItemRepository)
         {
-            _filterItemRepository = filterItemRepository;
+            FilterItemRepository = filterItemRepository;
```

Comment:
Why the name change here?

Reply:
Rider's naming convention suggestion, as `_filterItemRepository` would make it a private variable, whereas in fact it's a protected variable.